src/Coordination/KeeperStorage.cpp

```diff
@@ -1,7 +1,5 @@
 /// NOLINTBEGIN(clang-analyzer-optin.core.EnumCastOutOfRange)
 
-#include <iterator>
-#include <variant>
```

Comment:
Minor nitpick: Removing variant seems odd considering the variant was moved inside this file. It's probably ok because some other file has included it transitively, but in general I'd rather declare all the things I use inside the file.